pkgs/by-name/bk/bk/package.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchCrate
+, go-md2man
+, gnused
+, installShellFiles
+, lib
+, pkg-config
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bk";
+  version = "0.6.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-rSMvx/zUZqRRgj48TVVG7RwQT8e70m0kertRJysDY4Y=";
+  };
+
+  cargoHash = "sha256-pE5loMwNMdHL3GODiw3kVVHj374hf3+vIDEYTqvx5WI=";
+
+  nativeBuildInputs = [ go-md2man installShellFiles pkg-config ];
+
+  postBuild = ''
+    sed -i '$ a # Source and further info' README.md
```

Comment:
Any particular you don't just use `cat` or `echo`? I mean, sure it works with `sed`, it's just an unusual way of doing it.

Reply:
I believe because I thought I want/need to insert the lines somewhere in the middle, but then decided appending fits better.